.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 
 basestation.*
+.direnv/
```

Comment:
is this really necessary? I would rather see it in one's global .gitignore, no?

Reply:
Makes sense. I should put it in my global configuration. Thanks.